dev/com.ibm.ws.concurrent/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -52,6 +56,10 @@ defaultPriority.desc=Default priority for threads created by the managed thread
 jndiName=JNDI name
 jndiName.desc=JNDI name
 
+longRunningPolicy=Long Running Policy
+longRunningPolicy$Ref=Long running policy
+longRunningPolicy.desc=Concurrency policy for tasks that specify the LONGRUNNING_HINT execution property with value of true. If multiple executors specify the same concurrency policy then the policy's constraints are enforced across tasks submitted by the whole the collection of executors that specify the policy. If unspecified, the long running concurrency policy defaults to the concurrency policy for normal duration tasks. 
```

Comment:
should we put `true` in quotes or something?  To make it clear that translators should not translate the value in other languages.

Reply:
Proposed change for last sentence:
> If unspecified, the long running concurrency policy defaults to the value of `concurrencyPolicy`